hyperspy/io.py

```diff
@@ -188,6 +194,11 @@ def load(filenames=None,
         then square brackets are escaped before wildcard matching with
         ``glob.glob()``. If False, square brackets are used to represent
         character classes (e.g. ``[a-z]`` matches lowercase letters).
+    %s
+    %s Only used with ``stack=True``.
+    load_original_metadata : bool
+        If ``True``, all metadata will be added them to ``original_metadata``.
```

Comment:
If ``True``, all metadata contained in the input file will be added to ``original_metadata``.